mungegithub/github/github.go

```diff
@@ -1786,6 +1786,26 @@ func (obj *MungeObject) AddAssignee(owner string) error {
 	return nil
 }
 
+// IsAssignee will determine if the owner is a valid assignee in this repo
+func (obj *MungeObject) IsAssignee(owner string) (bool, error) {
+	_, response, err := obj.config.client.Issues.IsAssignee(
+		context.Background(),
+		obj.Org(),
+		obj.Project(),
+		owner,
+	)
+	if err != nil {
+		glog.Errorf("Error determining if user %v is a valid assignee: %v", owner, err)
+		return false, err
+	}
+	// For this method, 404 is marked as not an error
```

Comment:
Yeah, it's handled in [the `isAssignee` API call](https://github.com/kubernetes/test-infra/blob/master/vendor/github.com/google/go-github/github/issues_assignees.go#L47) by calling [`parseBoolResponse`](https://github.com/kubernetes/test-infra/blob/master/vendor/github.com/google/go-github/github/github.go#L662-L679)